bylaws/009-per-workflow.md

```diff
@@ -0,0 +1,85 @@
+
+Unless a proposal is moved to Draft stage again, it must remain in Review stage for a minimum of four weeks and until two independent viable trial implementations can be demonstrated. The responsibility for finding such trial implementations and presenting them to the Core Committee lies with the Working Group, and especially the Editor and Sponsor. As not all specifications are PHP interfaces where the definition of "implementation" is self-evident, the Sponsor should use good faith judgement to determine when that is the case. Any member of the Core Committee may object to a given trial implementation as irrelevant or insufficient with due cause.
+
+Once four weeks have passed and two viable trial implementations can be demonstrated, the Sponsor may call an Acceptance Vote. If the Acceptance Vote fails, the specification may remain in Review.
+
+## Accepted
+
+If the Acceptance Vote passes, then the proposal officially becomes an accepted PER.  It is considered the 1.0.0 release of that PER.
+
+## Evolution
+
+Unlike a PSR, a PER is expected to evolve over time.  Once the initial version of a PER is approved, the Working Group remains active but its minimum requirements drop to an Editor and two additional individuals.  By default, the Working Group at time of first approval continues in that role.  The Working Group may continue to develop the PER over time as the context of the recommendation evolves, such as newly released PHP language versions, additional input from third parties, etc.
+
+For any proposed change to the PER, the Editor must notify the Core Committee via a post to the mailing list of an Intent to Merge a given change.
+
+If no member of the Core Committee objects within seven days, the proposed change is assumed accepted by default.  Any member of the Core Committee may request during that seven-day period that the change be put to a vote.  If any member does so, the change is immediately put to an Acceptance Vote to determine if it is accepted or not.
+
+The Editor is empowered to declare tagged versions of a PER at any time.  Tagged versions MUST follow Semantic Versioning conventions.  However, any change that would necessitate a new Major release MUST be put to an Acceptance Vote by the Core Committee.
+
+## Staffing Changes
+
+The Editor of an approved PER is empowered to add or remove members from the Working Group at any time, with notification to the Core Committee.
+
+Should the Editor of a PER Working Group resign or become inactive, the Core Committee may appoint a new Editor.  Preference will be given to the existing Working Group members at that time.
+
+## Abandoned
```

Comment:
Would an "Accepted" PER be marked as "Abandoned" if it's WG dissolves?  Or would it forever remain "Accepted"?

Reply:
The PER would remain accepted, I presume. I suppose we probably do want to have a process to officially deprecate and abandon a PER's artifacts.